Doc/whatsnew/3.11.rst

```diff
@@ -1884,6 +1884,17 @@ New Features
 Porting to Python 3.11
 ----------------------
 
+* Many macros have been converted to functions in the Python C API,
+  making them usable from other programming languages and avoiding
+  `macro pitfalls <https://gcc.gnu.org/onlinedocs/cpp/Macro-Pitfalls.html>`_
```

Comment:
No, only a few macros were converted to regular functions usable in other programming languages. My recent attempt to convert a few static inline functions to regular functions is not popular and might be blocked: https://github.com/python/cpython/pull/97017

Please remove "making them usable from other programming languages".

Reply:
Also it should read "converted to static inline functions" and +1 to Victor's comment. There is difference between static inline and regular function at ABI level.

Reply:
Applied all changes as requested; see [my other comment](https://github.com/python/cpython/pull/98315/files#r997349456) (couldn't make it as a suggestion here since it didn't cover the affected lines)